packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -145,87 +131,181 @@ export default {
 <style lang="scss" module>
 
 $font-sizes: (
-    fontSizeSmall: (
+    sm: (
         title: heading-s,
         text: body-s,
     ),
-    fontSizeMedium: (
+    md: (
         title: heading-m,
         text: body-l,
     ),
-    fontSizeLarge: (
+    lg: (
         title: heading-l,
         text: subheading-s,
     ),
-    fontSizeXLarge: (
+    xl: (
         title: heading-xl,
         text: subheading-s,
     ),
-    fontSizeXXLarge: (
+    xxl: (
         title: heading-xxl,
         text: subheading-l,
     )
 );
 
+$flexMargin: spacing(x3);
```

Comment:
I'd call this `$mediaElement-content-margin`

Reply:
yep will make the change.